src/test/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/pdl/VegadresseTest.kt

```diff
@@ -0,0 +1,73 @@
+package no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class VegadresseTest {
+
+    @Test
+    fun avstandTilAnnenAdresse() {
```

Comment:
`Skal returnere false hvis avstand til annen adresse er mindre enn minimumsavstand for automatisk behhandling`